doc/src/guides/solving/solving-guidance.md

```diff
@@ -0,0 +1,221 @@
+<sympy.solvers.solvers.nsolve>` instead, for example
+
+```py
+>>> from sympy import nsolve, cos
+>>> from sympy.abc import x
+>>> nsolve(cos(x) - x, x, 2)
+0.739085133215161
+```
+
+If you receive non-closed-form solutions such as {class}`CRootOf
+<sympy.polys.rootoftools.ComplexRootOf>`, you can evaluate them numerically
+using {func}`evalf n <sympy.core.evalf.EvalfMixin.n>`:
+
+```py
+>>> from sympy import solve
+>>> from sympy.abc import x
+>>> solutions = solve(x**5 - x - 1, x, dict=True)
+>>> print(solutions)
+[{x: CRootOf(x**5 - x - 1, 0)}, {x: CRootOf(x**5 - x - 1, 1)}, {x: CRootOf(x**5 - x - 1, 2)}, {x: CRootOf(x**5 - x - 1, 3)}, {x: CRootOf(x**5 - x - 1, 4)}]
+>>> [solution[x].n(3) for solution in solutions]
+[1.17, -0.765 - 0.352*I, -0.765 + 0.352*I, 0.181 - 1.08*I, 0.181 + 1.08*I]
+```
+
+({class}`CRootOf <sympy.polys.rootoftools.ComplexRootOf>` represents an indexed
+complex root of a polynomial.)
+
+## Use Exact Values
+
+If you want to preserve the exact mathematical values of symbols such as
+transcendental numbers and [square roots](symbolic-computation), define them so
+that SymPy can interpret them symbolically, for example use SymPy's
+{class}`.Pi`:
+
+```py
+>>> from sympy import symbols, solve, pi
+>>> x = symbols('x')
+>>> solve(x**2 - pi, x, dict=True)
+[{x: -sqrt(pi)}, {x: sqrt(pi)}]
+```
+
+If you use the standard Python math version of $\pi$, Python will pass that
+inexact value to SymPy, leading to an inexact, numerical solution:
+
+```py
+>>> from sympy import symbols, solve
+>>> from math import pi
+>>> x = symbols('x')
+>>> solve(x**2 - pi, x, dict=True)
+[{x: -1.77245385090552}, {x: 1.77245385090552}]
+```
+
+To use exact values for numbers such as $6.2$ or $1/2$, refer to [Python numbers
+vs. SymPy Numbers](python-vs-sympy-numbers).
+
+In certain cases, using an inexact value will prevent SymPy from finding a
+result. For example, this exact equation can be solved:
+
+```py
+>>> from sympy import symbols, solve, sqrt
+>>> x = symbols('x')
+>>> eq = x**sqrt(2) - 2
+>>> solve(eq, x, dict=True)
+[{x: 2**(sqrt(2)/2)}]
+```
+
+but if you use the inexact equation `eq = x**1.4142135623730951 - 2`, SymPy will
+not return a result despite attempting for a long time. 
+
+## Options That Can Speed up {func}`~.solve`
+
+### Include Solutions Making Any Denominator Zero by Using `check=False`
+
+Normally, {func}`~.solve` checks whether any solutions make any denominator
+zero, and automatically excludes them. If you want to include those solutions,
+and speed up {func}`~.solve` (at the risk of obtaining invalid solutions), set
+`check=False`:
+
+```py
+>>> from sympy import Symbol, sin, solve
+>>> x = Symbol("x")
+>>> solve(sin(x)/x, x, dict=True) # 0 is excluded
+[{x: pi}]
+>>> solve(sin(x)/x, x, dict=True, check=False) # 0 is not excluded
+[{x: 0}, {x: pi}]
+```
+
+### Do Not Simplify Solutions by Using `simplify=False`
+
+Normally, {func}`~.solve` simplifies all but polynomials of order 3 or greater
```

Comment:
Unless I'm mistaken, `simplify` is about simplifying the solutions, not the equations. I guess this is copied from the `solve` docstring. Maybe @smichr can confirm either way. The "polynomials of order 3" thing is an unnecessary detail to include here IMO.

Reply:
Oh I didn't read the rest of the sentence. So I guess it does both. Either way, I would just omit the technical details here, which can be found in the solve docstring. Just mention that simplify=False makes it not try to simplify the equation and solutions, which can sometimes slow things down if they are complicated.